### PR TITLE
feat : avatar added

### PR DESCRIPTION
**Pull request by sach10-create**
Avatar added into the component library.

### CODE REVIEW

File: components/avatar/avatar.html [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Nova UI</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css"
+        integrity="sha512-Fo3rlrZj/k7ujTnHg4CGR2D7kSs0v4LLanw2qksYuRlEzO+tcaEPQogQ0KaoGN26/zrn20ImR1DfuLWnOo7aBA=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="./src/css/simple-avatar.css">
+    <!-- <link rel="stylesheet" href="../../css/components.css" /> -->
```
Comment (sonishreyas) [inline components/avatar/avatar.html]: This can be removed

**Review by sonishreyas — COMMENTED** [review]
Looks good to me

File: components/avatar/avatar.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+                    </div>
+                </div>
+            </section>
+        </main>
+        <nav>
+            <a class="burger-nav"></a>
+            <h2>Components</h2>
+            <ul class="list-components">
+                <a href="" class="navbar-link">
+                    <li class="component">Alerts</li>
```
Comment (itsmebipul00) [inline components/avatar/avatar.html:77]: You could instead wrap the a tag inside the li tag

**Comment by itsmebipul00** [comment]
Great work btw!!

File: components/avatar/src/css/profile-avatar.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
+@import url(../../../../css/component.css);
+
+body {
+    background-color: var(--background-color);
+    font-size: 2rem;
+}
+
+.avatar {
+    border-radius: 50%;
+    padding: 0 0.5rem 0 0.5rem;
```
Comment (iRohitGaur) [inline components/avatar/src/css/profile-avatar.css]: try to give in px to values less than 10px

File: components/avatar/src/css/simple-avatar.css [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  height: 6rem;
+  width: 6rem;
+  font-size: 1.5rem;
+}
+
+.avatar-text {
+  display: flex;
+  flex-wrap: wrap;
+  /* justify-content: center; */
+  /* align-items: center; */
```
Comment (iRohitGaur) [inline components/avatar/src/css/simple-avatar.css]: remove commented code

File: css/root-style.css [hunk 5]
```diff
@@ -4,51 +4,56 @@
...
-    --shadow-color : #9ca3af;
+    --font-color-light: #000000;
+
+    --online: green;
+    --away: orange;
+    --offline: red;
+    --white: #ffffff;
+    --black: #000000;
+    --off-white: #d4d4d8;
+    --shadow-color: #9ca3af;
```
Comment (iRohitGaur) [inline css/root-style.css:21]: suggestion: keep variable names in uppercase for better visibility

**Review by iRohitGaur — APPROVED** [review]
Good to merge with changes 👍🏻

File: css/badge-icon.css [hunk 6]
```diff
@@ -1,46 +1,45 @@
...
+  width: 4rem;
+  height: 4rem;
+}
+
+.icon-container {
+  background-color: var(--off-white);
+  padding: 1rem;
+  border-radius: 1rem;
+  border: 2px solid var(--off-white);
+  box-shadow: 0.5rem 0.5rem 0.5rem var(--shadow-color);
```
Comment (zoef-yt) [inline css/badge-icon.css]: use px here

File: css/code-tabs.css [hunk 7]
```diff
@@ -1,83 +1,83 @@
...
-    border-bottom: 0.25rem solid var(--primary-color);
-  }
-  
-  .code {
-    font-weight: bold;
-  }
+  display: flex;
+  flex-direction: column;
+  /* background: var(--secondary-background-color);  */
+  /* background: #00b2b2; */
```
Comment (zoef-yt) [inline css/code-tabs.css]: remove commented code.

File: css/profile-avatar.css [hunk 8]
```diff
@@ -1,33 +1,33 @@
...
-  .s-img {
-    width: 6rem;
-  }
+  background-color: var(--background-color);
+  font-size: 2rem;
+}
+
+.avatar {
+  border-radius: 50%;
+  padding: 0 0.5rem 0 0.5rem;
```
Comment (zoef-yt) [inline css/profile-avatar.css]: use px for rem less than 1rem

File: css/root-style.css [hunk 9]
```diff
@@ -4,51 +4,56 @@
...
-    --secondary-background-color-dark:#FFFAFA;
-    --font-color-dark : #ffffff;
+    --secondary-background-color-dark: #FFFAFA;
+    --font-color-dark: #ffffff;
     /* --tabs-background-color : #f3f4f6; */
 
     /* Light theme */
     /* --primary-color-light : #FF6122; */
-    --primary-color-light : #4F7777;
+    /* --primary-color-light : #4F7777; */
```
Comment (zoef-yt) [inline css/root-style.css]: removed the commented code if not in use.

File: css/simple-avatar.css [hunk 10]
```diff
@@ -1,49 +1,51 @@
 @import url(../../../../css/component.css);
 /* @import url(../../../../css/root-style.css); */
```
Comment (zoef-yt) [inline css/simple-avatar.css]: remove commented code

**Review by zoef-yt — CHANGES_REQUESTED** [review]
There are a lot of commented code make sure you remove it

**Pull request merged**